apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -440,7 +443,10 @@ schema("/nodes/:node/bridges/:id/:operation") ->
             ],
             responses => #{
                 204 => <<"Operation success">>,
-                400 => error_schema('BAD_REQUEST', "Forbidden operation, bridge not enabled"),
+                400 => error_schema(
+                    'BAD_REQUEST',
+                    "Problem with configuration of external service or bridge not enabled"
```

Comment:
Can one be more specific about what the configuration problem might be?

Reply:
I'm afraid that's a limitation of the schema descriptions (OpenAPI/swagger?), it doesn't allow for a list of possible outcomes (or does it?). That's why I tried to describe the nature of possible messages. The actual error message is different and rather specific.